release_docs/RELEASE.txt

```diff
@@ -47,6 +47,12 @@ New Features
 
     Configuration:
     -------------
+    - Autotools builds now build the szip filter by default when an appropriate
+      library is found
+
+      Since libaec is prevalent and BSD-licensed for both encoding and
+      decoding, we build the szip filter by default now.
+
```

Comment:
This is true for CMake, but looking at the code in configure.ac and configure I believe that for autotools using a found szip will require a prebuilt libsz.  If found it will build it into HDF5, but I don't see that autotools will build it from source.

Reply:
That's what I thought.  Perhaps - 

    Autotools builds now include the szip filter by default when and appropriate library is found.

    Since libaec is prevalent and BSD-licensed for both encoding and decoding, the default behavior for both CMake 
    and Autotools builds will  include an appropriate and available szip library.  CMake will download and build the 
    libaec szip library by default.

Another question - what if HDF5 finds the old szip with license restrictions for encoding?